include/itkMorphologicalContourInterpolator.hxx

```diff
@@ -88,8 +88,8 @@ protected:
   MorphologicalContourInterpolatorParallelInvoker() {}
 
 private:
-  virtual void
-  ThreadedExecution( const DomainType& subDomain, const ThreadIdType threadId )
+  void
+  ThreadedExecution( const DomainType& subDomain, const ThreadIdType threadId ) ITK_OVERRIDE
```

Comment:
Same as above.

Reply:
override implies virtual.  There is no need to retain the virtual when using override.  Newer versions of clang warn about this redundancy.